_includes/community-tutorial-navigation.html

```diff
@@ -0,0 +1,15 @@
+<div class="text-center py-3 border-top border-bottom my-3">
+{%- if page.previousTutorial %}
+{%- assign previousTutorial = '/tutorials/' | append: page.previousTutorial.link %}
+<i class="fas fa-angle-left fa-lg me-2"></i>
+Previous tutorial:
+<a href="{{ previousTutorial | relative_url}}">{{page.previousTutorial.title}}</a>
+{%- endif %}
+| <a href="/tutorials/community_tutorials">Community tutorials main page</a> |
+{%- if page.nextTutorial %}
+Next tutorial:
+{%- assign nextTutorial = '/tutorials/' | append: page.nextTutorial.link %}
+<a href="{{ nextTutorial | relative_url}}">{{ page.nextTutorial.title }}</a>
+<i class="fas fa-angle-right fa-lg ms-2"></i>
+{%- endif %}
+</div>
```

Comment:
this looks like a lot of duplication, maybe we can implement the option for a landing page also in the main one?

Reply:
I don't quite understand what you're aiming at but overall community tutorials look good. Possibly one of the missing things is a link to community tutorials directly from main menu (Getting started)

Reply:
@bedroesb if you see something is still missing, please go ahead and make the modifications